mattermost-bot/src/main/java/band/effective/core/HEICToPNGInterceptor.kt

```diff
@@ -0,0 +1,38 @@
+package band.effective.core
+
+import okhttp3.Interceptor
+import okhttp3.Response
+import okhttp3.ResponseBody
+import android.graphics.BitmapFactory
+import android.graphics.Bitmap
+import okhttp3.MediaType
+import java.io.ByteArrayOutputStream
+
+class HEICToPNGInterceptor : Interceptor {
```

Comment:
What about interceptor for coil or decoder? If lib's decoder not work, try write custom decoder like in lib